doc/source/tep/TEP14.md

```diff
@@ -17,7 +17,7 @@
 Introduction & motivation
 =========================
 
-This TEP is a spin-off of [TEP3]. It proposes a refactoring of taurus.core to get rid of some tango-centric implementation details that were left by [TEP3]. The objective is to obtain a cleaner and more scheme-agnostic API for creation of new schemes. Several inter-related aspects of the taurus core are refactored:
+This TEP is a spin-off of [TEP3](https://sourceforge.net/p/tauruslib/wiki/TEP3/). It proposes a refactoring of taurus.core to get rid of some tango-centric implementation details that were left by [TEP3](https://sourceforge.net/p/tauruslib/wiki/TEP3/). The objective is to obtain a cleaner and more scheme-agnostic API for creation of new schemes. Several inter-related aspects of the taurus core are refactored:
```

Comment:
Changes applied in most of the URLs. For the SEPs, and some of the taurus URLs, the URLs are still the old ones pointing to sourceforge. They can be corrected in the near future, once those URLs exists on github or on taurus-scada.org.